cdk_stacks/README.md

```diff
@@ -108,6 +109,12 @@ LLMEndpointStack
 (.venv) $ cdk deploy --require-approval never LLMEndpointStack
 ```
 
+#### Step 6: Deploy the Streamlit app on ECS Fargate
```

Comment:
Could you comment the Step 6 is optional?
I write this sample code as simple as possible. 
I would like to run Streamlit App on SageMaker Studio itself for reducing the complexity.
Also, there are many options to deploy Streamlit App on AWS.
Therefore, I would like you make the Step 6 optional. 
So, the sample code can be working even without deploying Streamlit App on ECS.